docs/backend/models.md

```
@@ -362,20 +336,14 @@ Always paginate large result sets:
 page = 1
 size = 20
 offset = (page - 1) * size
-galleries = db.query(Gallery)\
-    .filter_by(owner_id=user_id)\
-    .offset(offset)\
-    .limit(size)\
-    .all()
+galleries = db.query(Gallery).filter_by(owner_id=user_id).offset(offset).limit(size).all()
 ```
 
 ### Counter Updates
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)